tests/pipelines/pixart/test_pixart.py

```diff
@@ -120,7 +120,6 @@ def test_save_load_optional_components(self):
             "generator": generator,
             "num_inference_steps": num_inference_steps,
             "output_type": output_type,
-            "mask_feature": False,
```

Comment:
@patrickvonplaten with the current code changes, even if the `mask_feature` argument is set to True as long as the initial embeds are provided, this code path is not going to get hit:

https://github.com/huggingface/diffusers/blob/69996938cf7b41d4a69aea7aa6ab08a9b76a115d/src/diffusers/pipelines/pixart_alpha/pipeline_pixart_alpha.py#L256

Earlier it wasn't getting hit improperly. Let me know if that makes sense.

Reply:
ok!